cmd/rocsp-tool/main.go

```diff
@@ -118,31 +157,365 @@ func main2() error {
 		return fmt.Errorf("'issuers' section of config JSON is required.")
 	}
 	clk := cmd.Clock()
-	client, err := rocsp_config.MakeClient(&c.ROCSPTool.Redis, clk)
+	redisClient, err := rocsp_config.MakeClient(&c.ROCSPTool.Redis, clk)
 	if err != nil {
 		return fmt.Errorf("making client: %w", err)
 	}
 
-	for _, respFile := range flag.Args() {
-		err := storeResponse(respFile, issuers, client, clk)
+	var db *sql.DB
+	var ocspGenerator capb.OCSPGeneratorClient
+	if c.ROCSPTool.LoadFromDB != nil {
+		lfd := c.ROCSPTool.LoadFromDB
+		db, err = configureDb(&lfd.DB)
+		if err != nil {
+			return fmt.Errorf("connecting to DB: %w", err)
+		}
+
+		ocspGenerator, err = configureOCSPGenerator(lfd.GRPCTLS,
+			c.ROCSPTool.LoadFromDB.OCSPGeneratorService, clk, metrics.NoopRegisterer)
+		if err != nil {
+			return fmt.Errorf("configuring gRPC to CA: %w", err)
+		}
+		setDefault(&lfd.Speed.RowsPerSecond, 2000)
+		setDefault(&lfd.Speed.ParallelSigns, 100)
+	}
+
+	if len(flag.Args()) < 1 {
+		helpExit()
+	}
+
+	ctx := context.Background()
+	cl := client{
+		issuers:       issuers,
+		redis:         redisClient,
+		db:            db,
+		ocspGenerator: ocspGenerator,
+		clk:           clk,
+	}
+	switch flag.Args()[0] {
+	case "store":
+		for _, respFile := range flag.Args() {
```

Comment:
This loop appears to still be assuming that every non-flag argument is a filename; now that it'll be called as `./bin/rocsp-tool store file1 file2 file3` that's no longer true. Need to skip the zeroth non-flag arg.
```suggestion
		for _, respFile := range flag.Args()[1:] {
```

I think this is also an argument in favor of splitting this out to its own function, and having `main2` simply say
```
case "store":
  storeResponsesFromFiles(flag.Args())
```
for easier testing and detecting things like this.